### PR TITLE
[release-v1.11] Trust manager integration 

### PR DESCRIPTION
**Pull request by pierDipi**

### CODE REVIEW

**Comment by pierDipi** [comment]
``

/go/src/github.com/openshift-knative/serverless-operator/go.mod:5: unknown directive: toolchain
```

This needs https://github.com/openshift-knative/serverless-operator/pull/2451

File: config/openshift-trusted-cabundle.yaml [hunk 1]
```diff
@@ -21,3 +21,4 @@ metadata:
     app.kubernetes.io/version: devel
     app.kubernetes.io/name: knative-eventing
     config.openshift.io/inject-trusted-cabundle: "true"
+    networking.knative.dev/trust-bundle: "true"
```
Comment (matzew) [inline config/openshift-trusted-cabundle.yaml:24]: is this just extra `label`, so that Knative can use the injected bits in addition ?

Comment (pierDipi) [reply config/openshift-trusted-cabundle.yaml:24]: Yes, this is to move away from custom patches eventually

**Comment by pierDipi** [comment]
```
INFO[2024-01-26T08:34:00Z] Setting the cluster to Cincinnati instance: https://api.integration.openshift.com/api/upgrades_info/graph
{"component":"entrypoint","file":"k8s.io/test-infra/prow/entrypoint/run.go:169","func":"k8s.io/test-infra/prow/entrypoint.Options.ExecuteProcess","level":"error","msg":"Process did not finish before 2m0s timeout","severity":"error","time":"2024-01-26T08:29:04Z"}
```
/retest

**Comment by pierDipi** [comment]
@matzew @Cali0707 all tests are passing now, I had to create the bundle on installation as TLS bundle tests are running in regular jobs as it's not behind feature flag

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**